### ISSUE TITLE
Make colors constant for EmotionalState enum

### ISSUE THREAD

**Issue by WJXHenry** [CMPUT301F19T22/3OwO1#63]
**Description:** Find a way to obtain the color resource from the file or define a new constants file. Currently unable to access the android color resource from the `EmotionalState` class.

**Rationale:** The colors are currently hard-coded in the `EmotionalState` enum.

**Comment by bojotamara** [comment]
Yeah if we can't get it from the resource we could have like a color constants class for this or something
